soccer/gameplay/tactics/defense.py

```diff
@@ -84,8 +84,10 @@ def should_clear_ball(self):
 
         #Returns true if our robot can reach the ball sooner than the closest opponent
         safe_to_clear = False
-        if main.ball().pos.mag() < constants.Field.ArcRadius * 2 and main.ball(
-        ).vel.mag() < .75 and not evaluation.ball.is_in_our_goalie_zone():
+        if (abs(main.ball().pos.x) < constants.Field.PenaltyLongDist and 
```

Comment:
This change actually conflicted with the defense rewrite, so it might have to be partially re-written. If you merge master now, you'll have to resolve a conflict, but it shouldn't be too bad.